static/css/core-override.css

```diff
@@ -90,8 +95,19 @@ section p {
     flex: none;
   }
   header nav.shortcuts {
-    flex: none
+    flex: none;
+    align-items: baseline;
+  }
+
+  article > aside .menu ul {
+    border-left: 1px dashed #d4d8de;
+    border-bottom: 1px dashed #d4d8de;
+    padding-left: 15px;
+    color: #6e7477;
```

Comment:
I see what this was trying to do, I think. But I don't believe it is succeeding?  The rule 'article > aside .menu .dd-item' seems to take care of it?  

Or maybe 'm not actually clear on what this is going for.

Reply:
im trying to get the dashed border to appear when you click the dropdown on the inactive items, currently it only shows up after you click on the link